src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,184 @@
+
+      # Whether the integration should run in verbose mode or not. Defaults to false.
+      verbose: false
+
+      # Whether the integration should run in audit mode or not. Defaults to false.
+      # Audit mode logs the uncompressed data sent to New Relic. Use this to log all data sent.
+      # It does not include verbose mode. This can lead to a high log volume, use with care.
+      audit: true
+
+      # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
+      # scrape_timeout: "30s"
+
+      # Length in time to distribute the scraping from the endpoints.
+      scrape_duration: "5s"
+
+      # Number of worker threads used for scraping targets.
+      # For large clusters with many (&gt;400) endpoints, slowly increase until scrape
+      # time falls between the desired `scrape_duration`.
+      # Increasing this value too much will result in huge memory consumption if too
+      # many metrics are being scraped.
+      # Default: 4
+      # worker_threads: 4
+
+      # Whether the integration should skip TLS verification or not. Defaults to false.
+      insecure_skip_verify: true
+
+    timeout: 10s
+
+``` 
+
+</Collapser>
+
+<Collapser
+    id="step3"
+    title="Step 3. Forward Ansible controller logs to New Relic`"
+    >
+
+You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent) to forward Ansible Automation Platform logs to New Relic. 
+
+Your log file is present in:
+/etc/newrelic-infra/logging.d/
+
+If you don't see your log file in the path listed above, then you will need to create a log file. 
+
+To create the log file, add the script below to the logging.yml file:
+
+```
+  - name: ansible-tower.log
+    file: /var/log/tower/tower.log
+    attributes:
+      logtype: ansible_tower_log
+
+```
+</Collapser>
+
+ <Collapser
+    id="step4"
+    title="Step 4. Restart your infrastructure agent"
+    >
+
+Before you can start reading your data use the instructions in our [infrastructure agent docs](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) to restart your infrastructure agent. 
+</Collapser>
+
+<Collapser
+    id="step5"
+    title="Step 5. Use NRQL to view your data"
+    >
+You can use this NRQL query to view your Ansible Automation Platform data:
+
+```
+FROM Metric SELECT latest(awx_pending_jobs_total) AS 'Pending Jobs', latest(awx_running_jobs_total) AS 'Running Jobs'
+```
+
+</Collapser>
+</CollapserGroup>
+
+
+## See Ansible Automation Controller metrics in a dashboard [#dash]
+
+With only the infrastructure agent installed and instrumented with your app, you can view your raw data in our Metrics & events. Our default dashboards transform that raw data into charts and graphs, which provide a bird's eye view of your system's health. To install our default dashboards, go to our Ansible Automation Controller instant observability page.
```

Comment:
@homelessbirds I'm confused by the word "only" here. How about leaving it out?

I'm not sure what this sentence means: "With only the infrastructure agent installed and instrumented with your app...." Could we remove "only"? Not sure what that adds. What is your "app"? Are we talking about Ansible Automation Platform? Also, you'll need to put bold around "Metrics and events" since it's a page in the UI. Better yet, get a link to that and insert an entire URL so users don't need to poke around. 

Also, I'd recommend you insert a link directly to the instant observability page. This is under "Add data" in the UI.